.github/workflows/nightly.yml

```diff
@@ -40,3 +40,7 @@ jobs:
         run: npm publish --tag next --workspace=packages/ckbtc
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+      - name: Publish rosetta-client
```

Comment:
can we uncomment the CI part until the lib is ready?

otherwise we won't be able to publish any new version of ic-js

Reply:
good point, I'll do that.

Reply:
For now I'll rever these changes